test/API.spec.js

```diff
@@ -0,0 +1,14 @@
+const template = require('../packages/oc-template-handlebars');
+const compiler = require('../packages/oc-template-handlebars-compiler');
```

Comment:
I don't know. Initially that was what I had. But as I moved tests out of the package folder and have all the test for all the template (both the base APIs and the compiler APIs) then it was way more natural to just have a single one. I guess that if in the future those tests grows, then we could pheraps split them